src/components/OneTimeCodeInput/index.tsx

```diff
@@ -0,0 +1,148 @@
+import React from 'react'
+import styled from '@emotion/styled'
+import { fieldStyles, Help, LabelText } from '../Input'
+import { space } from '../../theme'
+import { useRef, useState } from 'react'
+import { VisuallyHidden } from '../VisuallyHidden'
+
+interface OneTimeCodeContainerProps {
+  length: number
+}
+
+interface OneTimeCodeInputProps {
+  length: number
+  id: string
+  hideLabel?: boolean
+  ariaLabel: string
+  label?: string
+  disabled?: boolean
+  helpText?: string
+  onChange?: (value: any) => void
+}
+
+const OneTimeCodeContainer = styled.section<OneTimeCodeContainerProps>`
+  display: grid;
+  grid-template-columns: repeat(${props => props.length}, 1fr);
+  grid-gap: ${space[8]};
```

Comment:
Nit: Empty line between the properties and selector

Reply:
Actually, why don't you use the `text-align` in the styled `Input` itself?

Reply:
@glenngijsberts The `Input` is made with the `fieldStyles` from the `Input` component. If I add it there it won't work because the basic `fieldStyles` will override it and force it left again.

Reply:
@bobbonius ehhhhh, if you declare it like this:

```js
const Input = styled.input`
  ${fieldStyles};
  text-align: center;
`
```

![Screenshot 2021-11-16 at 11 40 45](https://user-images.githubusercontent.com/14276144/141970538-4ef7fc28-b064-4217-a2a4-3e449dde6ce0.png)

![Screenshot 2021-11-16 at 11 41 05](https://user-images.githubusercontent.com/14276144/141970586-eab34fce-b8af-4548-b08e-4795732f92e4.png)

Then the `text-align: center` will overwrite the text-align coming from the `fieldStyles` right? Did you try that?